src/service.js

```diff
@@ -565,6 +582,44 @@ function disposeServiceInternal(id, service) {
 
 
 /**
+ * Whether the specified service implements `EmbeddableService` interface.
+ * @param {!Object} service
+ * @return {boolean}
+ */
+export function isEmbeddable(service) {
+  return typeof service.adoptEmbedWindow == 'function';
+}
+
+
+/**
+ * Asserts that the specified service implements `EmbeddableService` interface
+ * and typecasts the instance to `EmbeddableService`.
+ * @param {!Object} service
+ * @return {!EmbeddableService}
+ */
+export function assertEmbeddable(service) {
+  dev().assert(isEmbeddable(service),
```

Comment:
Yes. I meant to say it directly calls `.adtopEmbedWindow` on the services themselves.